packages/core/src/sass/account-wizard.scss

```diff
@@ -445,3 +445,146 @@
         line-height: 1.5;
     }
 }
+
+.trading-assessment {
+    display: flex;
+    flex-direction: column;
+
+    @include mobile() {
+        height: calc(100vh - 160px);
+        overflow: auto;
+    }
+
+    &__wrapper__dropdown--list--display {
+        @include mobile() {
+            height: 5.5rem !important;
+        }
+    }
+
+    &__side-note {
+        text-align: justify;
+        padding: 0 7rem;
+
+        @include mobile() {
+            padding: 0 1.6rem;
+        }
+    }
+
+    &__header {
+        margin-top: 2.4rem;
+        width: -webkit-fill-available;
```

Comment:
I wouldn't advise to use webkit properties since they're not supported by all the browsers

Reply:
Will change this